app/views/categories/index.html.erb

```diff
@@ -0,0 +1,32 @@
+<%= render 'header' %>
+<main class="categories">
+  <div class="credit-info">
+    <p class="gift-text">Total Amount</p>
+    <p class="gift-amt"><%= Deal.where(author_id: current_user.id).sum(:amount) %>
+    </p>
+  </div>
+  <div class="categories-container">
+    <div class="category-scroll">
+      <% if @categories.present? %>
+        <% @categories.each do |category| %>
+          <%= link_to user_category_path(current_user, category) do %>
+            <div class="category-container">
+              <%= image_tag url_for(category.icon.variant(resize: "100x100")) if category.icon.attached? %>
+              <div class="cat-name-date">
+                <h2 class="cat-name"><%= category.name %></h2>
+                <p class="cat-date"><%= category.created_at %></p>
+              </div>
+              <span class="cat-total-amt"><%= category.deals.sum(:amount) %></span>
+            </div>
+          <% end %>
+        <% end %>
+      <% else %>
+        <p class="no-resource-msg">No categories available</p>
+      <% end %>
+    </div>
+  </div>
+  <div class="category-actions">
+  <%= link_to "Add a new category", new_user_category_path(current_user), class: 'new-category-btn' %>
+  </div>
+</main>
```

Comment:
- [ ] Kindly work on the design and layout of the Add a new category button to match the design given. The button is placed at the bottom of the screen in fixed CSS positioning styles. Please modify the styles. 👍🏽
- [ ] Fron the original design below, the text Scan, on the green button can be changed to Add a new category. And please implement the same on the Transaction page. This is the idea of how your design should look like. 

**Your Design**

<img width="453" alt="Screenshot 2024-02-10 at 6 21 57 PM" src="https://github.com/ometman/budget-app/assets/50941074/e731aa44-e574-44c6-bd78-e01c41011cfd">

**The Original Design**

<img width="345" alt="Screenshot 2024-02-10 at 6 32 28 PM" src="https://github.com/ometman/budget-app/assets/50941074/b802777d-7178-4c9f-823e-6e0f9bc8daa7">